pkgs/tools/misc/ical2orgpy/default.nix

```diff
@@ -1,31 +1,41 @@
-{ lib, python3Packages, fetchPypi, ... }:
+{ lib
+, python3
+, fetchFromGitHub
+}:
 
-python3Packages.buildPythonPackage rec {
+python3.pkgs.buildPythonApplication rec {
   pname = "ical2orgpy";
-  version = "0.4.0";
+  version = "0.5";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "sha256-7/kWW1oTSJXPJtN02uIDrFdNJ9ExKRUa3tUNA0oJSoc=";
-  };
+  format = "setuptools";
 
-  disabled = python3Packages.pythonOlder "3.9";
+  src = fetchFromGitHub {
```

Comment:
Why is that preferred over `fetchPypi`?

Reply:
Personal preference. It's closer to the source and guaranteed to e.g. contain all test files.